tests/js/fixtures/sentryAppComponent.js

```diff
@@ -3,7 +3,19 @@ export function SentryAppComponent(params = {}) {
     uuid: 'ed517da4-a324-44c0-aeea-1894cd9923fb',
     type: 'issue-link',
     schema: {
-      link: {required_fields: [{type: 'text', name: 'a', label: 'A'}]},
+      link: {
+        required_fields: [
+          {type: 'text', name: 'a', label: 'A', default: 'issue.title'},
+          {type: 'textarea', name: 'c', label: 'C', default: 'issue.description'},
+          {
+            type: 'select',
+            name: 'numbers',
+            label: 'Numbers',
+            options: [{label: 'one', value: 1}, {label: 'two', value: 2}],
+            defaultValue: 2,
```

Comment:
I think the way we should denote a default is the same way we do when we fetch options. That is, one of the objects in the `options` array should specify `default: true`.

We should also reflect this in our schema.

Reply:
I was going off https://github.com/getsentry/sentry/blob/master/src/sentry/mediators/external_requests/select_requester.py#L87 so maybe the correct thing to do is to actually have it be `choices` instead of `options`?

Reply:
Well, this is mimicking the schema a developer will write, not how we reformat that for the front-end. So it should follow https://www.notion.so/sentry/Component-Select-b54547c31f6448e2af6b87457307b0bd, which says you write `options` if it's a static list.